### DISCUSSION TITLE
how to theme table correctly ?

### DISCUSSION THREAD

**Discussion by orenmizr** [element-plus/element-plus#4352] [Q&A]
**""element-plus": "^1.2.0-beta.3",**
**can someone show the correct way of using themes? the idea is the change the css variables i assume not override with scss vars with new colors.** 

```
$--font-path: "element-plus/packages/theme-chalk/src/fonts";

@forward "element-plus/theme-chalk/src/common/var.scss" with (
  $colors: (
    "primary": (
      "base": red,
      
    ),
  ),
  $table: (
    'border-color': var(--el-border-color-lighter),  <------------------ need to changes the variables, not the the table
    'border': 1px solid var(--el-table-border-color),
    'font-color': var(--el-text-color-regular),
    'header-font-color': var(--el-text-color-secondary),
    'row-hover-background-color': var(--el-background-color-base),
    'current-row-background-color': var(--el-color-primary-light-9),
    'header-background-color': #202328,
    'fixed-box-shadow': 0 0 10px rgba(0, 0, 0, 0.12),
    'background-color': var(--el-color-white),
    'tr-background-color': #2f343b,
    'expanded-cell-background-color': var(--el-color-white),
  )

);
      
      
@import "element-plus/theme-chalk/src/index.scss";
```

**Comment by msidolphin** [comment]
please upgrade to the latest version, refer to the [documentation](https://element-plus.org/en-US/guide/theming.html#how-to-override-it)

**Reply by orenmizr** [reply]
what has changed ? can you please elaborate. is this a better way... or just a fix.

**Reply by msidolphin** [reply]
You can override the css var
```css
:root {
   --el-border-color-lighter: red;
}
```
 we may provide new solution (config-provide) in the release version

**Reply by orenmizr** [reply]
where are listed... i see var.scss... where are the lists of the css3 variables that the scss uses ?

**Reply by orenmizr** [reply]
fount it. some are located in _var.scss. 
sadly, there isn't a real good way to customize the theme to dark. you can't fully change the css3 variables to affect all you need. and you can't change components cross site because they depend on those css3 vars. 

i hope it will be fixed in next version. i do believe this ui lib is better than anything else i've seen. and tested a lot lately.

**Reply by emojiiii** [reply]
#3754
